src/Sylius/Bundle/AttributeBundle/Resources/views/attributeChoice.html.twig

```diff
@@ -10,9 +10,9 @@
                     <div class="list-group row">
```

Comment:
Sure, these are things that have to be fixed :+1: I'm sure I can handle these issues in the nearest future.